services/api/lib/queries.js

```diff
@@ -1,6 +1,10 @@
 module.exports = {
-  findSkittleById: "SELECT id, color, user_id FROM skittles WHERE id = $1;",
-  createSkittle: "INSERT INTO skittles (color, user_id) VALUES($1, $2);",
-  updateSkittleById: "UPDATE skittles SET color = $1 WHERE id = $2",
-  deleteSkittleById: "DELETE FROM skittles WHERE id = $1"
+  users: {
+    create: "INSERT INTO users (username, language, country) VALUES($1, $2, $3) RETURNING id;",
+    find: "SELECT id, username, created_at, updated_at, language, country, moderator, staff " +
+      "FROM users WHERE deleted_at IS NULL AND id = $1;",
+    update: "UPDATE users SET(username, language, country, updated_at) = " +
+      "($1, $2, $3, CURRENT_TIMESTAMP) WHERE deleted_at IS NULL AND id = $4 RETURNING username, language, country;",
```

Comment:
Because we are keying off `deleted_at` we should add an index in the creation SQL.